ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -700,20 +718,47 @@ private void uploadClusterAdminInfos(Map<String, Map<String, Map<String, String>
    * @param adminConfigZNodePath the znode path of admin config.
    */
   private void deleteClusterAdminInfos(String clusterAdminType, String adminConfigZNodePath) {
-    Properties storeProps = new Properties();
-    storeProps.setProperty("helix.property.store.root.path", "/" + clusterName + "/" + PROPERTYSTORE_STR);
-    HelixPropertyStoreConfig propertyStoreConfig = new HelixPropertyStoreConfig(new VerifiableProperties(storeProps));
```

Comment:
Thanks for simplifying this. Could you also make the change for `uploadClusterAdminInfos` method as well (line 701 ~ 705)?

Reply:
done